numba/cuda/tests/cudapy/test_dispatcher.py

```diff
@@ -412,16 +412,16 @@ def simple_lmem(ary):
         # Check that getting the local memory per thread for all signatures
         # provides the same values as getting the shared mem per block for
         # individual signatures.
-        sh_mem_f32_all = simple_lmem.get_local_mem_per_thread()
-        sh_mem_f64_all = simple_lmem.get_local_mem_per_thread()
-        self.assertEqual(sh_mem_f32_all[sig_f32.args], local_mem_f32)
-        self.assertEqual(sh_mem_f64_all[sig_f64.args], local_mem_f64)
+        local_mem_f32_all = simple_lmem.get_local_mem_per_thread()
+        local_mem_f64_all = simple_lmem.get_local_mem_per_thread()
+        self.assertEqual(local_mem_f32_all[sig_f32.args], local_mem_f32)
+        self.assertEqual(local_mem_f64_all[sig_f64.args], local_mem_f64)
```

Comment:
```suggestion
        local_mem_all = simple_lmem.get_local_mem_per_thread()
        self.assertEqual(local_mem_all[sig_f32.args], local_mem_f32)
        self.assertEqual(local_mem_all[sig_f64.args], local_mem_f64)
```
Is this just the same dictionary being assigned to different variables, in which case the above should be equivalent?

Reply:
Whoops, yes.